packages/product-json-to-csv/src/main.js

```diff
@@ -217,12 +217,34 @@ export default class ProductJsonToCsv {
 
     if (!channelIds.length && !customerGroupIds.length) return prices
 
+    const CHUNK_SIZE = 150
     // resolve channels by their ids
-    const channelsById: Object = await this._getChannelsById(channelIds)
+    let channelsById: Object = {}
+    if (channelIds.length > CHUNK_SIZE) {
+      channelsById = await chunk(channelIds, CHUNK_SIZE).reduce(
```

Comment:
I would just move this into the `_get*` so that the logic is a bit "hidden" for the `resolvePriceReferencesThingy`